[Bug]: FOSSA CI job fails because it cannot find the latest version

### Meltano Version

NA

### Python Version

NA

### Bug scope

Other

### Operating System

NA

### Description

This is causing failures for:
- https://github.com/meltano/meltano/pull/6553
- https://github.com/meltano/meltano/pull/6505

It will likely cause CI failures for any new PRs until it is resolved.

Example: https://github.com/meltano/meltano/runs/7639031548

```
fossas/fossa-cli info checking GitHub for latest tag
fossas/fossa-cli crit unable to find '' - use 'latest' or see https://github.com/fossas/fossa-cli/releases for details
Error: Process completed with exit code 1.
```

### Code

_No response_

--- a/.github/workflows/dependency-review.yml
+++ b/.github/workflows/dependency-review.yml
@@ -10,7 +10,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  FOSSA_CLI_INSTALLER_VERSION: '3.3.10'
+  FOSSA_CLI_VERSION: '3.3.11'
 
 permissions:
   contents: read
@@ -29,8 +29,7 @@ jobs:
 
       - name: FOSSA dependency license check
         run: |
-          # `$FOSSA_CLI_INSTALLER_VERSION` only controls the version of the installer used - the latest version of `fossa-cli` will always be used.
-          curl --no-progress-meter -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/v${FOSSA_CLI_INSTALLER_VERSION}/install-latest.sh | bash
+          curl --no-progress-meter -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash -s -- "v$FOSSA_CLI_VERSION"
 
           echo '## FOSSA dependency license check' >> $GITHUB_STEP_SUMMARY
           echo '' >> $GITHUB_STEP_SUMMARY

